### DISCUSSION TITLE
Add support for grouping up organization repositories / Allow pinning private repos

### DISCUSSION THREAD

**Discussion by mnj** [community/community#5759] [Other Feature Feedback, Questions, & Ideas]
We really need something like Gitlabs group feature, where you can group repositories in nested groups, to get a much better overview of which repos an organization have, just having a long flat list of repos, isn't super useful, when you start having hundreds of repos.

I know you can assign repos to a team in an organization, but that isn't what we are after they suffer from the same problem.

This is mostly a feature to get an overview, not to act as a way to manage repos (so unrelated to permissions etc), basically just a nested list of group names, that can have links to the repos.

And in addition to this, it could be nice if you could pin private repos in an organization, instead of only having the 6 public repos available to be pinned, then you could sortof (badly) fix the first point with a readme that links to all the private repos, although a proper group feature would be better.

**Comment by huehnerlady** [comment]
@mnj I agree with your point, would be super useful to have grouping. we 'missuse' topics for that so we can distinguish it a bit better. maybe a workaround for you too for now?

**Comment by nbrugger-tgm** [comment]
Simmilar request : https://github.com/github/feedback/discussions/7354

I would not limit it to organisations. There is imo no good reason to disable this for user profile level repos

**Comment by jason-dour** [comment]
Absolutely need this capability when navigating large private/internal corporate org structures.

**Comment by clecap** [comment]
I also support the pinning of private repositories. 

According to support: "It is currently only possible to pin public repositories and gists as the intention is to allow viewers of your publicly accessible profile page to see them."

This is fine. However:

1. There is no disadvantage by allowing a repository owner to structure her repository in the way she believes it is reasonable.
2. It would greatly help structuring ones repository the way one needs to ones *own* demands.

**Comment by pawelcern** [comment]
I am wondering if anybody from GitHub considered feature discussed. I am going to host several projects on GitHub. Each consists of several repositories - hardware, linux distribution for application processor, microcontroller firmware etc. Flat structure is complicated to navigate / manage.
